src/main/resources/com/google/api/codegen/php/main.snip

```diff
@@ -486,8 +486,17 @@
             {@""} * @@param array $optionalArgs {
              *     Optional.
             @join param : optionalFields
-                @join commentLine : context.php.getDocLinesWithPrefixes(context.getDescription(param), optionalParamDocLinePrefix(param), "          ")
-                    {@commentLine}
+                @if isPageSizeField(param, methodConfig)
+                    {@""} *     @@type integer ${@methodConfig.getPageStreaming.getPageSizeField.getSimpleName}
```

Comment:
So it looks like we're completely tossing away the proto-specified doc for page_size parameters. Is this what we intend? I think this is the first instance of us auto-generating API-specific behavioral documentation.